databuilder/setup.py

```diff
@@ -76,7 +76,8 @@
 
 rds = [
     'sqlalchemy>=1.3.6,<1.4',
-    'mysqlclient>=1.3.6,<3'
+    'mysqlclient>=1.3.6,<3',
+    'cx_Oracle==8.2.1'
```

Comment:
I think rds is meant for something else, not collection of libs regarding all rds systems. you can just create separate oracle group

Reply:
Done!